---
source: tests/output.rs
expression: "format!(\"{:?}\", miette::Error::new(tc.unwrap_err()))"
---
  [31mÃ—[0m Testcase did not run successfully
[31m  â•°â”€â–¶ [0m  [31mÃ—[0m An argument was of the wrong type
[31m      [0m   â•­â”€[[36;1;4mtest.kdl:3:17[0m]
[31m      [0m [2m2[0m â”‚             testcase {
[31m      [0m [2m3[0m â”‚                 foobar not_a_number
[31m      [0m   Â· [35;1m                â”€â”€â”€â”¬â”€â”€[0m[33;1m â”€â”€â”€â”€â”€â”€â”¬â”€â”€â”€â”€â”€[0m
[31m      [0m   Â·                    [35;1mâ”‚[0m         [33;1mâ•°â”€â”€ [33;1mthis one[0m[0m
[31m      [0m   Â·                    [35;1mâ•°â”€â”€ [35;1mThis node has an argument of a wrong kind[0m[0m
[31m      [0m [2m4[0m â”‚             }
[31m      [0m   â•°â”€â”€â”€â”€
[31m      [0m[36m  help: [0mThis verb takes a 'integer' as its argument here.
[31m      [0m
